test/integration/narrow/app/import/process-backlog.test.js

```diff
@@ -0,0 +1,91 @@
+const { buildDog, buildPerson, warmUpCache, validateAndInsertPerson, validateAndInsertDog, getBacklogRows } = require('../../../../../app/import/backlog-functions')
+jest.mock('../../../../../app/import/backlog-functions')
+const { dbLogErrorToBacklog } = require('../../../../../app/lib/db-functions')
+jest.mock('../../../../../app/lib/db-functions')
+const { getMicrochipType } = require('../../../../../app/lookups')
+jest.mock('../../../../../app/lookups')
+const { testBacklogPerson } = require('./persons')
+const { testBacklogDog } = require('./dogs')
+const backlogRows = require('./mock-backlog-rows')
+const { process } = require('../../../../../app/import/process-backlog')
+
+describe('ProcessBacklog test', () => {
+  test('Should return zero stats when no rows', async () => {
+    getMicrochipType.mockImplementation(() => Promise.resolve({ id: 9 }))
```

Comment:
Could you replace these `mockImplementation` calls with `mockResolved`?

Just to make the tests a bit more readable.

Reply:
I did this for all but the throw error tests since didn't seem an easy way (that worked)